CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   * `-ingester.max-global-series-per-metric`
 * [FEATURE] Flush chunks with stale markers early with `ingester.max-stale-chunk-idle`. #1759
 * [FEATURE] EXPERIMENTAL: Added new KV Store backend based on memberlist library. Components can gossip about tokens and ingester states, instead of using Consul or Etcd. #1721
+* [FEATURE] Allow Query Frontend to log slow queries #1744
```

Comment:
Could you mention the flag here, see the flush chunks with stale markers above.

Reply:
added flag.